tests/deal_ticket/test_stop_order.py

```diff
@@ -91,11 +91,12 @@ def test_submit_stop_order_rejected(continuous_market, vega: VegaService, page:
     page.get_by_test_id(submit_stop_order).click()
     page.get_by_test_id(stop_orders_tab).click()
 
-    vega.wait_fn(1)
+    vega.wait_fn(10)
```

Comment:
```suggestion
    vega.wait_fn(1)
```

reverting these changes as they are being skipped